workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectSystemHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Packages.com.improbable.gdk.core.GameObjectRepresentation.GameObjectCreation;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static class GameObjectSystemHelper
+    {
+        public static IEntityGameObjectCreator EntityGameObjectCreator;
```

Comment:
why is this needed here? can't it stay inside the system that it was in before?

Reply:
It needs to be static, because the way users give us this creator is currently in a bootstrap-style GameObject in the scene, and it seemed slightly nicer to put it in the static class at that point, esp. because users actually interact with this static class in the SystemConfig already, while the GameObjectInitializationSystem could be considered an implementation detail they shouldn't need to know about.

(The restructure moved that system into the feature module while keeping the creator in user code, so MonoBehaviour workflow users don't need to touch ECS code. That's also why the creator is now given nicely wrapped SpatialOSEntity instances)

Reply:
This class itself was originally created to split the stuff out from CoreSystemHelper btw where feature module systems don't belong, moving the EntityGameObjectCreator reference here was a bit of an afterthought.

Reply:
I understand the reason behind this class, it just doesn't feel like it's in the right place here

Reply:
Will think about where to put it